### ISSUE TITLE
curls copied and imported of graphql over http are not importing correctly. 

### ISSUE THREAD

**Issue by mmcguff** [a85/POSTMan-Chrome-Extension#12349]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

When you copy a curl of a http POST request where the body type is graphql, the curl generated is unable to be imported back into postman without throwing an error.  Here is short gif of the issue: 

### Steps To Reproduce

1.  Start with any graphql request over http.  In the example I am using the unauthenticated SpaceX graphql API.  

2.  Ensure the call is working.   

3.  copy the curl of the working request using the snippet option in postman

4.  Import that copied curl and see that running that once running request fails with a parsing error of some kind.  

The error looked like this: 

{
    "errors": [
        {
            "message": "Unknown operation named \"\".",
            "extensions": {
                "code": "OPERATION_RESOLUTION_FAILURE"
            }
        }
    ]
}

Below is the postman generated curl which will not import correctly: 

```bash
curl --location 'https://spacex-production.up.railway.app' \
--header 'Content-Type: application/json' \
--data '{"query":"query getCompanyData {\r\n    company {\r\n        ceo\r\n    }\r\n}","variables":{}}'
```

### Screenshots or Videos

![postman-graphql-over-http-import-issue](https://github.com/postmanlabs/postman-app-support/assets/10658437/47986c0e-4e50-4b1d-aa1f-28800d27e22a)

### Operating System

Windows

### Postman Version

10.18.6

### Postman Platform

Postman App

### User Account Type

Signed In User

### Additional Context?

The issue in postman web appears slightly different but it still doesn't work.  

Having CURL fail like this is big deal for my API Documentation development.  DX (Developer Experience) is a big deal for use case of postman for API Documentation.  If this issue is not resolved right away, I will be forced to use another provide for my API Documentation needs.

**Comment by VShingala** [comment]
@mmcguff Thanks for reporting the issue! We are working on the fix for this and will inform here once this is fixed.

**Comment by jnorrisdod** [comment]
Is there a fix for this yet? Having this issue and PM says "You are up to date! Postman v10.20.10 is the latest version."

**Comment by VShingala** [comment]
@mmcguff @jnorrisdod This issue has been fixed with the latest release v10.22. I've also confirmed it to be working as expected. Feel free to open this thread if you face any further related issue.
